src/errors.rs

```diff
@@ -8,23 +8,23 @@
 use quick_xml::error::Error as XmlError;
 
 error_chain! {
-    types {
-        Error, ErrorKind, Result;
-    }
+//    types {
+//        Error, ErrorKind, ResultExt, Result;
+//    }
```

Comment:
Could you remove commented lines (it is confusing as `ResultExt` won't mean anything).